src/app/shared/post-capture-card/post-capture-card.component.ts

```diff
@@ -52,13 +56,39 @@ export class PostCaptureCardComponent implements OnInit {
   constructor(
     private readonly diaBackendAssetRepository: DiaBackendAssetRepository,
     private readonly translocoService: TranslocoService,
-    private readonly imageStore: ImageStore
+    private readonly imageStore: ImageStore,
+    private readonly bottomSheet: MatBottomSheet
   ) {}
 
   ngOnInit() {
     this._transaction$.next(this.transaction);
   }
 
+  openOptionsMenu() {
+    const bottomSheetRef = this.bottomSheet.open(OptionsMenuComponent);
+    bottomSheetRef
+      .afterDismissed()
+      .pipe(
+        tap((option?: Option) => {
+          if (option === Option.Delete) {
+            this.remove();
+          } else if (option === Option.Share) {
+            this.share();
+          }
+        }),
+        untilDestroyed(this)
+      )
+      .subscribe();
+  }
+  remove() {
+    return this.asset$
+      .pipe(
+        first(),
+        concatMap(asset => this.diaBackendAssetRepository.remove$(asset))
```

Comment:
After you remove the `DiaBackendAsset`, you need to call `DiaBackendTransactionRepository.refresh$()` to update the PostCapture wall.